packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -169,6 +237,81 @@ const MT5TradeScreen: FC<MT5TradeScreenProps> = ({ mt5Account }) => {
                     </div>
                 </div>
 
+                {platform === ctraderPlatform && (
+                    <Fragment>
+                        <ul className='wallets-mt5-trade-screen__ctrader-account-list'>
+                            {sortedCtraderAccountList?.map(account => (
+                                <li
+                                    className='wallets-mt5-trade-screen__ctrader-account-list__item'
+                                    key={account.login}
+                                >
+                                    <Text size='sm'>{account.login}</Text>
+                                    <Text size='sm' weight='bold'>
+                                        {account.display_balance}
+                                    </Text>
+                                </li>
+                            ))}
+                        </ul>
+                        {availableCtraderAccounts?.[0].available_count !== undefined &&
+                            availableCtraderAccounts[0].available_count > 0 && (
+                                <button
+                                    className='wallets-mt5-trade-screen__ctrader-get-more-button'
+                                    onClick={onClickCtraderGetMoreButton}
+                                    type='button'
+                                >
+                                    <span className='wallets-mt5-trade-screen__ctrader-get-more-button__icon'>
+                                        <LabelPairedPlusMdFillIcon fill='#000000' />
+                                    </span>
+                                    <Text size='xs'>Get another cTrader account</Text>
```

Comment:
Should we localize this page?

Reply:
@hasan-deriv also ensure that long texts are start-aligned for RTL support

Reply:
@lubega-deriv We should control that from `Text` component.